dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.mpCompatible-6.0.feature

```diff
@@ -2,7 +2,7 @@
 symbolicName=io.openliberty.mpCompatible-6.0
 visibility=private
 singleton=true
--features=com.ibm.websphere.appserver.eeCompatible-10.0
+-features=com.ibm.websphere.appserver.eeCompatible-10.0; ibm.tolerates:="11.0"
```

Comment:
There should not be a tolerates here, MP 6.0 is fixed to EE 10

Reply:
well some of them will also run with EE 11 if the spec is not updated.  This is temporary in order to get some of the MP dependencies in jakarta features to work correctly with EE 11.

Reply:
I removed it.